.travis.yml

```diff
@@ -1,10 +1,21 @@
 language: php
 
 php:
+  - 7.1
+  - 7.0
   - 5.6
   - 5.5
   - 5.4
   - 5.3
+  - nightly
+  - hhvm
+  - hhvm-nightly
```

Comment:
... what is just temporary until Travis updates its infrastructure - something that will happen sooner rather than later, Ubuntu Precise's end-of-life is in about 6 months.